packages/kbn-securitysolution-utils/src/esql/get_index_list_from_esql_query.ts

```diff
@@ -5,13 +5,14 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-import { getIndexPatternFromESQLQuery } from '@kbn/esql-utils';
+import { getIndexPatternFromESQLQueryDeprecated } from '@kbn/esql-utils';
 
 /**
  * parses ES|QL query and returns array of indices
  */
+// this should be changed to use the new getIndexPatternFromESQLQuery function
 export const getIndexListFromEsqlQuery = (query: string | undefined): string[] => {
-  const indexString = getIndexPatternFromESQLQuery(query);
+  const indexString = getIndexPatternFromESQLQueryDeprecated(query);
```

Comment:
ℹ️ Same here. We need to use the async getIndexPatternFromESQLQuery instead but I prefer the code owners to handle it